SmartDeviceLink/NSMutableDictionary+Store.m

```diff
@@ -24,30 +24,48 @@ - (nullable id)sdl_objectForName:(SDLName)name {
     return self[name];
 }
 
+- (nullable SDLEnum)sdl_enumForName:(SDLName)name {
+    return [self sdl_objectForName:name ofClass:NSString.class];
+}
+
+- (nullable NSArray<SDLEnum> *)sdl_enumsForName:(SDLName)name {
+    return [self sdl_objectsForName:name ofClass:NSString.class];
+}
+
 - (nullable id)sdl_objectForName:(SDLName)name ofClass:(Class)classType {
```

Comment:
Currently we don't have test cases to check getting data that has a different data type than the expected data type. Can you *NSMutableDictionary+StoreSpec.m* file to test the `sdl_objectForName: ofClass:` and `sdl_objectsForName:ofClass:` methods to make sure the right data type is being returned.

Reply:
@NicoleYarroch Implemented in 5ee2ac0